sklearnex/covariance/incremental_covariance.py

```diff
@@ -196,6 +198,45 @@ def _onedal_partial_fit(self, X, queue=None, check_input=True):
 
         return self
 
+    @wrap_output_data
+    def score(self, X_test, y=None):
+        xp, _ = get_namespace(X_test)
+
+        location = self.location_
+        if sklearn_check_version("1.0"):
+            X = self._validate_data(
+                X_test,
+                dtype=[np.float64, np.float32],
+                reset=False,
+                copy=self.copy,
+            )
+        else:
+            X = check_array(
+                X_test,
+                dtype=[np.float64, np.float32],
+                copy=self.copy,
+            )
```

Comment:
We are not supporting skl <1.0

Reply:
Lets try and keep that discussion to #1906